modules/service/src/main/resources/db/migration/V0885__cfp_limits.sql

```diff
@@ -0,0 +1,132 @@
+-- Drop the existing RA/Dec limits and replace them with non-nullable
+-- site-specific limits.  The limits will be calculated if not explicitly
+-- specified.
+DROP VIEW v_cfp;
+
+ALTER TABLE t_cfp
+  ADD COLUMN c_north_ra_start  d_angle_µas,
+  ADD COLUMN c_north_ra_end    d_angle_µas,
+  ADD COLUMN c_north_dec_start d_angle_µas,
+  ADD COLUMN c_north_dec_end   d_angle_µas,
+  ADD COLUMN c_south_ra_start  d_angle_µas,
+  ADD COLUMN c_south_ra_end    d_angle_µas,
+  ADD COLUMN c_south_dec_start d_angle_µas,
+  ADD COLUMN c_south_dec_end   d_angle_µas;
+
+UPDATE t_cfp
+   SET c_north_ra_start  = COALESCE(c_ra_start,             0),
```

Comment:
Didn't know about `COALESCE`, is it so that we basically see the old value but have a default if null?

Reply:
Right, it chooses the first non-null value.